tox.ini

```diff
@@ -21,16 +21,16 @@ usedevelop = True
 commands = {posargs:py.test -s -x -f -v}
 
 [testenv:docs]
-description = invoke sphinx-build and try to build the HTML page, and check that URIs are valid
+description = invoke sphinx-build to build the HTML docs, check that URIs are valid
 basepython = python
 deps = sphinx >= 1.6.3, < 2
        towncrier >= 17.8.0
        {[testenv]deps}
 passenv = http_proxy https_proxy no_proxy SSL_CERT_FILE
-commands = sphinx-build -d .tox/docs_doctree doc .tox/docs_out --color -W -bhtml
-           sphinx-build -d .tox/docs_doctree doc .tox/docs_out --color -W -blinkcheck
+commands = sphinx-build -d {toxworkdir}/docs_doctree doc {toxworkdir}/docs_out --color -W -bhtml
+           sphinx-build -d {toxworkdir}/docs_doctree doc {toxworkdir}/docs_out --color -W -blinkcheck
 
-[testenv:style]
+[testenv:fix-lint]
```

Comment:
I expect a change to `appveyor.yml` (where this gets currently run) as well

Reply:
heh in fact this is a sneaky way to *skip* the style checks: https://ci.appveyor.com/project/RonnyPfannschmidt/tox/build/1.0.444/job/mfdxp0fvjcy67g94